matecheck.py

```diff
@@ -36,14 +36,20 @@ def analyze_fens(self, fens):
 if __name__ == "__main__":
     freeze_support()
     parser = argparse.ArgumentParser(
-        description="Check how many (best) mates an engine finds in e.g. matetrack.epd.",
+        description="Check how many (best) mates an engine finds in e.g. matetrack.epd. By default this script takes the (subjective) point of view of most AB engines: reporting a mate score for the side to move means this is currently the best line it has found, and in general it can guarantee to achieve the reported result, but there may be better lines. In particular, reporting a losing mate in a drawn or won position may happen by design, and so is not considered a mistake.",
```

Comment:
I inserted `in general` here precisely because for negative mate scores SF does not seem to be able to survive for at least that many moves. The same probably holds for other AB engines, too.